packages/liveblocks-core/src/room.ts

```diff
@@ -1624,47 +1626,28 @@ export function createRoom<
     delegates.authenticate()
   );
 
-  async function httpPostToRoom(
-    endpoint: "/send-message" | "/text-metadata",
-    body: JsonObject
-  ) {
-    return httpClient1.fetch(
-      endpoint === "/send-message"
-        ? url`/v2/c/rooms/${config.roomId}/send-message`
-        : url`/v2/c/rooms/${config.roomId}/text-metadata`,
-      {
-        method: "POST",
-        body: JSON.stringify(body),
-      }
-    );
-  }
-
   async function createTextMention(userId: string, mentionId: string) {
-    return httpClient1.fetch(url`/v2/c/rooms/${config.roomId}/text-mentions`, {
-      method: "POST",
-      body: JSON.stringify({
-        userId,
-        mentionId,
-      }),
+    await httpClient1.rawPost(url`/v2/c/rooms/${config.roomId}/text-mentions`, {
+      userId,
+      mentionId,
```

Comment:
I intentionally did not introduce a behavioral change here. For each of these `.rawGet()`, `.rawPost()`, etc calls we currently do, we should ideally convert those to `.get()`, `.post()` etc on a case-by-case basis, and be sure that it's okay that those calls would throw the response in case of an error. If the call site is not expecting a throw, it may be breaking change, that's why I kept this around for now.

Ideally we convert them all, and then remove the `.rawXxx()` methods from the client.